### ISSUE TITLE
Support for other keyboard layouts?

### ISSUE THREAD

**Issue by samscott89** [rchatterjee/pam-typopw#13]
Does the adaptive typo-tolerance work for a non-US/non-standard layout? If so, is this detected automatically, or is there a configuration parameter to set somewhere? (The target use uses US/dvorak layout).

**Comment by rchatterjee** [comment]
No, not yet. I added the layout of Dvorak but does not do anything right now. I got a similar request for Russian keyboard. The distance calculation that needs to take keyboard layout in the consideration is in a different repo, called [https://github.com/rchatterjee/word2keypress](word2keypress). But haven't got change to work that.
